cli/src/pcluster/config/cluster_config.py

```diff
@@ -2014,6 +2015,12 @@ def _register_validators(self, context: ValidatorContext = None):
                 compute_resource_name=compute_resource.name,
                 compute_resource_efa_enabled=compute_resource.efa.enabled,
             )
+            placement_group = self.get_chosen_placement_group_setting_for_compute_resource(compute_resource)
+            self._register_validator(
+                MultiAzPlacementGroupValidator,
+                multi_az_enabled=self.multi_az_enabled,
+                placement_group_enabled=placement_group.enabled or placement_group.assignment is not None,
```

Comment:
I think the PlacementGroup Resource should have a property to get the general status of whether a placement group was specified or not. This is probably a common use-case.

```
@property
def enabled_or_assigned(self):
    return self.enabled or self.assignment is not None
```

Reply:
Good catch.
I agree and apply the changes.